### ISSUE TITLE
bug: With --exclusive -i command it excludes all patch. It should include selected patches.bug in -i command.fine with official cli 3.1.1

### ISSUE THREAD

**Issue by dib12** [inotia00/ReVanced_Extended#1499]
### Type

Error while patching

### Tools used

CLI

### Application

 YouTube any version

### Bug description

With --exclusive -i command it excludes all patch. It should include selected patches

### Steps to reproduce

Add --exclusive -i  MicroG-support  and check 

### Relevant log output

```shell
~/revanced $ java -jar revanced-cli-3.1.4-all.jar patch yt182736.apk --custom-aapt2-binary aapt/aapt2 -m revanced-integrations-0.117.24.apk -b revanced-patches-2.190.24.jar --exclusive -i Default-video-quality -i Disable-Shorts-on-startup -i Enable-minimized-playback -i Enable-old-quality-layout -i Hide-cast-button -i Hide-channel-watermark -i Hide-end-screen-cards -i Hide-floating-microphone -i Hide-general-ads -i Hide-video-ads -i MicroG-support -i Spoof-player-parameters -i Swipe-controls -o yt_p_1827.apk
INFO: Loading patches
INFO: Setting patch options
INFO: Decoding app manifest
INFO: Add splash animation excluded by default
INFO: Alternative thumbnails excluded by default
INFO: Append time stamps information excluded by default
INFO: Bypass ambient mode restrictions excluded by default
INFO: Change homepage excluded by default
INFO: Custom branding YouTube name excluded by default
INFO: Custom branding icon MMT excluded by default
INFO: Custom branding icon Revancify blue excluded by default
INFO: Custom branding icon Revancify red excluded by default
INFO: Custom double tap length excluded by default
INFO: Custom package name excluded by default
INFO: Custom playback speed excluded by default
INFO: Custom seekbar color excluded by default
INFO: Custom speed overlay excluded by default
INFO: Default playback speed excluded by default
INFO: Default video quality excluded by default
INFO: Disable QUIC protocol excluded by default
INFO: Disable Shorts on startup excluded by default
INFO: Disable auto captions excluded by default
INFO: Disable haptic feedback excluded by default
INFO: Disable hdr video excluded by default
INFO: Disable landscape mode excluded by default
INFO: Disable pip notification excluded by default
INFO: Enable compact controls overlay excluded by default
INFO: Enable debug logging excluded by default
INFO: Enable external browser excluded by default
INFO: Enable minimized playback excluded by default
INFO: Enable new comment popup panels excluded by default
INFO: Enable new splash animation excluded by default
INFO: Enable new thumbnail preview excluded by default
INFO: Enable old quality layout excluded by default
INFO: Enable open links directly excluded by default
INFO: Enable seekbar tapping excluded by default
INFO: Enable tablet mini player excluded by default
INFO: Enable tablet navigation bar excluded by default
INFO: Enable wide search bar excluded by default
INFO: Force OPUS codec excluded by default
INFO: Force VP9 codec excluded by default
INFO: Force hide player button background excluded by default
INFO: Force premium heading excluded by default
INFO: Header switch excluded by default
INFO: Hide account menu excluded by default
INFO: Hide auto player popup panels excluded by default
INFO: Hide autoplay button excluded by default
INFO: Hide autoplay preview excluded by default
INFO: Hide button container excluded by default
INFO: Hide captions button excluded by default
INFO: Hide cast button excluded by default
INFO: Hide category bar excluded by default
INFO: Hide channel avatar section excluded by default
INFO: Hide channel watermark excluded by default
INFO: Hide collapse button excluded by default
INFO: Hide comment component excluded by default
INFO: Hide crowdfunding box excluded by default
INFO: Hide description components excluded by default
INFO: Hide double tap overlay filter excluded by default
INFO: Hide end screen cards excluded by default
INFO: Hide end screen overlay excluded by default
INFO: Hide feed flyout panel excluded by default
INFO: Hide filmstrip overlay excluded by default
INFO: Hide floating microphone excluded by default
INFO: Hide fullscreen panels excluded by default
INFO: Hide general ads excluded by default
INFO: Hide handle excluded by default
INFO: Hide info cards excluded by default
INFO: Hide latest videos button excluded by default
INFO: Hide layout components excluded by default
INFO: Hide load more button excluded by default
INFO: Hide mix playlists excluded by default
INFO: Hide music button excluded by default
INFO: Hide navigation buttons excluded by default
INFO: Hide navigation label excluded by default
INFO: Hide player button background excluded by default
INFO: Hide player flyout panel excluded by default
INFO: Hide player overlay filter excluded by default
INFO: Hide previous next button excluded by default
INFO: Hide quick actions excluded by default
INFO: Hide seek message excluded by default
INFO: Hide seekbar excluded by default
INFO: Hide shorts components excluded by default
INFO: Hide snack bar excluded by default
INFO: Hide suggested actions excluded by default
INFO: Hide suggested video overlay excluded by default
INFO: Hide suggestions shelf excluded by default
INFO: Hide time stamp excluded by default
INFO: Hide tooltip content excluded by default
INFO: Hide trending searches excluded by default
INFO: Hide video ads excluded by default
INFO: Language switch excluded by default
INFO: Layout switch excluded by default
INFO: MaterialYou excluded by default
INFO: MicroG support excluded by default
INFO: Overlay buttons excluded by default
INFO: Return YouTube Dislike excluded by default
INFO: Settings excluded by default
INFO: SponsorBlock excluded by default
INFO: Spoof app version excluded by default
INFO: Spoof player parameters excluded by default
INFO: Swipe controls excluded by default
INFO: Theme excluded by default
INFO: Translations excluded by default
INFO: Executing patches
INFO: Compiling modified dex files
INFO: Aligning yt182736.apk
```

### Solution

May be problem in this commit  https://github.com/inotia00/revanced-cli/commit/9228a2bfd93d2a62a37276c80d7fff2779157329

### Device Environment

Android 11 , revanced cli

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.
- [X] I have written the title and contents in English.
- [X] This issue does not reproduce on unpatched YouTube or YT Music.

**Comment by sams452m** [comment]
I'm also facing same issue there's issue with CLI for sure

**Comment by dib12** [comment]
May be problem in this commit https://github.com/inotia00/revanced-cli/commit/9228a2bfd93d2a62a37276c80d7fff2779157329

Waiting for @inotia00 response

**Comment by inotia00** [comment]
Support for CLI v4 is planned to be added within the next week
If possible, I will bump the official code without modifying it as much as possible

I think maybe that will solve the issue

**Comment by inotia00** [comment]
fixed in [revanced-cli-v4.0.2](https://github.com/inotia00/revanced-cli/releases/tag/v4.0.2)
